src/tests/ftest/util/dlck_utils.py

```diff
@@ -0,0 +1,85 @@
+"""
+  (C) Copyright 2025 Hewlett Packard Enterprise Development LP
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+from command_utils_base import CommandWithParameters, FormattedParameter
+from run_utils import run_remote
+
+
+class DlckCommand(CommandWithParameters):
+    """Defines the basic structures of dlck command."""
+
+    def __init__(self, server_host, path, pool_uuid=None, nvme_conf=None, storage_mount=None,
+                 verbose=True, timeout=None, sudo=True, env_str=None):
+        """Constructor that sets the common variables for sub-commands.
+
+        Args:
+            server_host (NodeSet): Server host to run the command.
+            path (str): path to the dlck command.
+            pool_uuid (str, optional): Pool UUID. Defaults to None.
+            nvme_conf (str, optional): NVMe config file path. Defaults to None.
+            storage_mount (str, optional): Storage mount point. Defaults to None.
+            verbose (bool, optional): Display command output in run.
+                Defaults to True.
+            timeout (int, optional): Command timeout (sec) used in run. Defaults to
+                None.
+            sudo (bool, optional): Whether to run dlck with sudo. Defaults to True.
+            env_str (str, optional): Environment variable string to prepend to command.
+        """
+        super().__init__("/run/dlck/*", "dlck", path)
+        # Pass environment variable string
+        self.env_str = ""
+        if env_str is not None:
+           self.env_str = str(env_str)
+
+        # We need to run with sudo.
```

Comment:
I hope this comment to become outdated very soon. 😄 
On the other hand it does not mean you will be able to run dlck in our CI as a normal user the next day.

I will keep this comment here for now. We will see where we will go with this.

Ref: https://daosio.atlassian.net/browse/DAOS-18324